swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/generator/core/utils/ClassUtils.java

```diff
@@ -24,6 +24,9 @@
 import java.util.List;
```

Comment:
seems that "io.servicecomb.swagger.generator.core.utils.ClassUtils.JAVA_RESERVED_WORDS" can be removed?